webpack.config.js

```diff
@@ -33,5 +35,11 @@ module.exports = {
   optimization: {
     minimize: false
   },
-  devtool: 'inline-source-map'
+  devtool: 'inline-source-map',
+  
+  performance: {
+    hints: false,
+    maxEntrypointSize: 500000,
```

Comment:
Is this getting around the warning about one the sample sizes? I thought we were going to modify the sample so it wasn't so big.